<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>startWindow</class>
 <widget class="QMainWindow" name="startWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>450</width>
    <height>350</height>
   </rect>
  </property>
  <property name="minimumSize">
   <size>
    <width>450</width>
    <height>350</height>
   </size>
  </property>
  <property name="maximumSize">
   <size>
    <width>450</width>
    <height>350</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>PokerTraining</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <layout class="QGridLayout" name="gridLayout_2">
    <item row="0" column="0">
     <widget class="QGroupBox" name="groupBox">
      <property name="title">
       <string>Game Settings</string>
      </property>
      <layout class="QGridLayout" name="_2">
       <item row="1" column="0">
        <widget class="QLabel" name="label_2">
         <property name="text">
          <string>Start Cash:</string>
         </property>
         <property name="buddy">
          <cstring>spinBox_startCash</cstring>
         </property>
        </widget>
       </item>
       <item row="1" column="1">
        <widget class="QSpinBox" name="spinBox_startCash">
         <property name="prefix">
          <string>$</string>
         </property>
         <property name="minimum">
          <number>1000</number>
         </property>
         <property name="maximum">
          <number>1000000</number>
         </property>
         <property name="singleStep">
          <number>50</number>
         </property>
         <property name="value">
          <number>2000</number>
         </property>
        </widget>
       </item>
       <item row="0" column="1">
        <widget class="QSpinBox" name="spinBox_quantityPlayers">
         <property name="minimum">
          <number>2</number>
         </property>
         <property name="maximum">
          <number>10</number>
         </property>
         <property name="value">
          <number>6</number>
         </property>
        </widget>
       </item>
       <item row="0" column="0">
        <widget class="QLabel" name="label">
         <property name="text">
          <string>Number of players:</string>
         </property>
         <property name="buddy">
          <cstring>spinBox_quantityPlayers</cstring>
         </property>
        </widget>
       </item>
       <item row="3" column="0">
        <widget class="QPushButton" name="pushButtonStartGame">
         <property name="maximumSize">
          <size>
           <width>337</width>
           <height>16777215</height>
          </size>
         </property>
         <property name="focusPolicy">
          <enum>Qt::NoFocus</enum>
         </property>
         <property name="text">
          <string>Start game</string>
         </property>
         <property name="icon">
          <iconset resource="resources/pokertraining.qrc">
           <normaloff>:/gfx/swbuttonlocalgame.png</normaloff>:/gfx/swbuttonlocalgame.png</iconset>
         </property>
         <property name="iconSize">
          <size>
           <width>37</width>
           <height>32</height>
          </size>
         </property>
        </widget>
       </item>
       <item row="2" column="0" colspan="2">
        <widget class="QGroupBox" name="groupBox_3">
         <property name="title">
          <string>Opponents profile</string>
         </property>
         <widget class="QRadioButton" name="radioButton_opponentsRandomlyChosen">
          <property name="geometry">
           <rect>
            <x>10</x>
            <y>100</y>
            <width>231</width>
            <height>18</height>
           </rect>
          </property>
          <property name="text">
           <string>Opponents randomly chosen</string>
          </property>
          <property name="checked">
           <bool>true</bool>
          </property>
         </widget>
         <widget class="QRadioButton" name="radioButton_opponentsLooseAggressive">
          <property name="enabled">
           <bool>true</bool>
          </property>
          <property name="geometry">
           <rect>
            <x>10</x>
            <y>40</y>
            <width>241</width>
            <height>18</height>
           </rect>
          </property>
          <property name="text">
           <string>Mostly loose-aggressive opponents</string>
          </property>
          <property name="checked">
           <bool>false</bool>
          </property>
         </widget>
         <widget class="QRadioButton" name="radioButton_opponentsTightAgressive">
          <property name="geometry">
           <rect>
            <x>10</x>
            <y>70</y>
            <width>231</width>
            <height>18</height>
           </rect>
          </property>
          <property name="text">
           <string>Mostly tight-aggressive opponents</string>
          </property>
         </widget>
        </widget>
       </item>
      </layout>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
  <action name="actionStartGame">
   <property name="icon">
    <iconset resource="resources/pokertraining.qrc">
     <normaloff>:/gfx/swbuttonlocalgame.png</normaloff>:/gfx/swbuttonlocalgame.png</iconset>
   </property>
   <property name="text">
    <string>Start Game ...</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+N</string>
   </property>
  </action>
  <action name="actionAboutPokerTraining">
   <property name="icon">
    <iconset resource="resources/pokertraining.qrc">
     <normaloff>:/gfx/system_help_64.png</normaloff>:/gfx/system_help_64.png</iconset>
   </property>
   <property name="text">
    <string>About PokerTraining</string>
   </property>
   <property name="toolTip">
    <string>About PokerTraining</string>
   </property>
  </action>
  <action name="actionConfigurePokerTraining">
   <property name="icon">
    <iconset resource="resources/pokertraining.qrc">
     <normaloff>:/gfx/systemsettings.png</normaloff>:/gfx/systemsettings.png</iconset>
   </property>
   <property name="text">
    <string>Configure PokerTraining ...</string>
   </property>
  </action>
  <action name="actionQuit">
   <property name="icon">
    <iconset resource="resources/pokertraining.qrc">
     <normaloff>:/gfx/application_exit.png</normaloff>:/gfx/application_exit.png</iconset>
   </property>
   <property name="text">
    <string>Quit</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+Q</string>
   </property>
  </action>
  <action name="actionAbout_Pokertraining">
   <property name="text">
    <string>About Pokertraining</string>
   </property>
  </action>
 </widget>
 <resources>
  <include location="resources/pokertraining.qrc"/>
 </resources>
 <connections>
  <connection>
   <sender>actionQuit</sender>
   <signal>triggered()</signal>
   <receiver>startWindow</receiver>
   <slot>close()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>-1</x>
     <y>-1</y>
    </hint>
    <hint type="destinationlabel">
     <x>221</x>
     <y>168</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
